flocker/node/agents/blockdevice.py

```diff
@@ -944,6 +944,42 @@ def get_device_path(blockdevice_id):
         """
 
 
+class IProfiledBlockDeviceAPI(IBlockDeviceAPI):
```

Comment:
I don't think needs to inherit from `IBlockDeviceAPI`. It can just be an additional interface a class implements in addition to `IBlockDeviceAPI`, with additional methods.